ChartsDemo-iOS/ChartsDemo-iOS.xcodeproj/project.pbxproj

```diff
@@ -1045,7 +1051,7 @@
 				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
 				PRODUCT_BUNDLE_IDENTIFIER = "com.dcg.$(PRODUCT_NAME:rfc1034identifier)";
-				PRODUCT_NAME = ChartsDemo;
+				PRODUCT_NAME = "ChartsDemo-iOS";
```

Comment:
did we changed the project strut, by purpose or accident?

Reply:
On purpose. Some naming was out of date, or inconsistent (macOS vs OS X).